src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs

```diff
@@ -80,12 +82,11 @@ private IGeoMasterClient InitClient()
             Dictionary<string, string> appSettings = new Dictionary<string, string>();
             foreach (var item in properties)
             {
-                if (AllowedlistAppSettingsStartingWith.Any(x => item.Key.StartsWith(x)))
+                if (AllowedlistAppSettingsStartingWith.Any(x => item.Key.StartsWith(x)) && !SensitiveAppSettingsEndingWith.Any(x => item.Key.EndsWith(x))
```

Comment:
Is `StartsWith` and `EndsWith` case agnostic? Just asking in case property keys in the future would be a different casing.

Reply:
While it is not a bad suggestion, it is highly unlikely that casing for some of these app settings will change in future just because a lot of code (Antares and outside Antares) has now a dependency on these settings ;-)

Reply:
> While it is not a bad suggestion, it is highly unlikely that casing for some of these app settings will change in future just because a lot of code (Antares and outside Antares) has now a dependency on these settings ;-)

Yea I was thinking the same too, its just one of those really small things that you don't want to bite you in the butt later  ¯\_(ツ)_/¯